frontend/src/pages/SuperAdmin.js

```diff
@@ -7,14 +7,14 @@ const SuperAdmin = () => {
   const { t } = useTranslation()
 
   return (
-    <>
+    <div className="col-9">
```

Comment:
Itt is jobb lenne a használat helyén beállítani az oszlopot, mert ott látszik, hogy mi veszi körül, hány egyéb oszlop van mellette.